client/components/section-nav/style.scss

```diff
@@ -347,6 +347,17 @@
 	}
 }
 
+.section-nav-tab__title {
```

Comment:
I've matched the current section convention in https://github.com/Automattic/wp-calypso/blob/63bc72ddb4f372b94efcd8f38dfa75bf8b86f32b/client/components/section-nav/style.scss - which doesnt match the CSS style guide, hence disabling the check. I thought it best to match what's there rather than have two different styles in the same code. I also didn't want to migrate all the old styles!